genesis/errors.go

```diff
@@ -130,3 +130,6 @@ var ErrNilQueryService = errors.New("nil query service")
 
 // ErrMissingElement signals a missing element event
 var ErrMissingElement = errors.New("missing element")
+
+// ErrGetVersionFromSC signals that a coll to "version" function on a contract resulted in an unexpected result
```

Comment:
"call" instead "coll"  and  "on a smart contract" instead "on a contract"

Reply:
fixed